### PR TITLE
[WIP] evm opcode assembly

### PR DESCRIPTION
**Pull request by kbrav**

### CODE REVIEW

**Comment by nmushegian** [comment]
pretty fucking cool if you ask me

**Comment by nmushegian** [comment]
I can't believe how bad this compiler output is

File: core/dmap.asm [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+INVALID
+LOG2
+PUSH5 0x6970667358
+0x22
+SLT
+KECCAK256
+SELFBALANCE
+GAS
+0x49
+DIFFICULTY
```
Comment (nmushegian) [inline core/dmap.asm:181]: w t f

File: core/dmap.asm [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+PUSH1 0x0
+RETURN
+INVALID
+LOG2
+PUSH5 0x6970667358
+0x22
+SLT
+KECCAK256
+SELFBALANCE
+GAS
```
Comment (nmushegian) [inline core/dmap.asm:179]: w t f is this shit
what are they putting in the compiler

Comment (kbrav) [reply core/dmap.asm:179]: Maybe swarm hash

Comment (kbrav) [reply core/dmap.asm:179]: And/or ipfs CID

File: test/bounds.js [hunk 3]
```diff
@@ -2,20 +2,20 @@ module.exports.bounds = {
   dmap: {
     get:  [25952, 25952],
     set: {
-      0: {0: [27959, 27959], 1: [68527, 68527]},
-      1: {0: [26848, 26848], 1: [28727, 28727], 2: [34327, 34327]},
```
Comment (nmushegian) [inline test/bounds.js]: lol epic

**Pull request closed without merging**